cwltool/pathmapper.py

```diff
@@ -246,3 +248,9 @@ def reversemap(self, target):  # type: (Text) -> Tuple[Text, Text]
             if v[1] == target:
                 return (k, v[0])
         return None
+
+    # On windows os.path.join would use backslash to join path, since we would use these paths in Docker we would convert it to /
+    def pathFix(self,path):  # type: (Text) -> (Text)
```

Comment:
hmm. I would change it. I prefer to move it to utils, as we are using it at one more file. what do you say. Yes a new name is needed. I will do that.